zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/Publisher.java

```diff
@@ -692,6 +703,25 @@ private static boolean publishDatasets(Collection collection) throws IOException
         return datasetsPublished;
     }
 
+    private static Future<Boolean> publishImages(Collection collection){
+        String collectionId = collection.getDescription().getId();
+
+        return pool.submit(() -> {
```

Comment:
Consider having a separate thread pool for publishing images / anything that isn't traditional publishing. If for some reason the images publish is failing and hanging, it could end up hogging all of the threads in the pool, preventing other publishes from taking place.

Some more explanation of the idea here https://stackoverflow.com/questions/26243422/is-having-a-single-threadpool-better-design-than-multiple-threadpools